server/routerlicious/packages/services-utils/src/throttlerMiddleware.ts

```diff
@@ -67,6 +68,7 @@ export function throttle(
 
         if (throttleOptions.weight === 0) {
             logger?.info("Throttle middleware created with 0 weight: Replacing with no-op middleware.");
+            Lumberjack.info("Throttle middleware created with 0 weight: Replacing with no-op middleware.");
```

Comment:
In Lumberjack, we use the property
```
[CommonProperties.telemetryGroupName]: "throttling"
```
to indicate that a log should not go to the default "routerlicious" or "historian" table, but instead to the throttling table. Something similar is done for Winston through the messageMetadata field.
```
messageMetaData: {
    ...
    eventName: "throttling",
},
```

It's true that `logger?.info` above did not specify such `messageMetaData`, making the log go to the default table. But I suspect that was an accident. Since this is throttlerMiddleware.ts, I think we might want all logs to go to the throttling table. Can you confirm that with @znewton and make updates if my theory is correct?

If it is correct, and we want to make sure all logs in this file go to the throttling table, we can even create a helper method, like "getBaseLumberjackProperties()" which could return:

```
{
                [CommonProperties.telemetryGroupName]: "throttling",
}
```
and then use it throughout the code like this:
```typescript
Lumberjack.info(
    "Throttle middleware created with 0 weight: Replacing with no-op middleware.",
    getBaseLumberjackProperties());
    
   Lumberjack.error(`Throttle increment failed`, {
                ...(getBaseLumberjackProperties()),
                [ThrottlingTelemetryProperties.key]: throttleId,
            }, error);

```

Reply:
Base on the message logged by the above `logger?.info`, it looks the execution is still in ThrottlerMiddleware and does not really get into Throttler yet, given message is saying no-op, and it looks no `messageMetaData` available at this point. So I guess that's the reason it wasn't logged into throttling table? Anyway, let's confirm with @znewton .

Agree with the error case comments and will update properly.

Reply:
Yes I think this was a mistake. I probably wanted it to go to the Throttler table. Thanks for catching!